pact/coin-contract/coin.pact

```diff
@@ -7,19 +7,22 @@
   \or issue the '(use coin)' command in the body of a module declaration."
 
 
+  (use coin-sig)
+
   ; --------------------------------------------------------------------------
   ; Schemas and Tables
 
   (defschema coin-schema
     balance:decimal
-    guard:guard
-    )
+    guard:guard)
   (deftable coin-table:{coin-schema})
 
-  (defschema creates-schema
-    exists:bool
+  ; the shape of a cross-chain transfer (used for typechecking)
+  (defschema transfer-*
```

Comment:
kind of a weird/unidiomatic name, I would probably call it `transfer-schema` ...

Reply:
Changed